tensorboard/program.py

```diff
@@ -556,4 +664,41 @@ def _fix_werkzeug_logging(self):
     logging.getLogger('werkzeug').setLevel(logging.NOTSET)
 
 
+class _ArgparseMonkeyPatch(object):
+  """Make Python 2.7 behave like Python 3 w.r.t. default subcommands.
+
+  The behavior of argparse was changed in Python 3.3. When a parser
```

Comment:
Might as well add a reference to https://bugs.python.org/issue16308 and https://github.com/python/cpython/commit/f97c59aaba2d93e48cbc6d25f7ff9f9c87f8d0b2 for the curious?